tests/data_context/migrator/test_cloud_migrator.py

```diff
@@ -53,6 +50,53 @@ def test__send_configuration_bundle_sends_valid_http_request(
     )
 
 
+@pytest.mark.unit
+@pytest.mark.cloud
+def test__send_validation_results_sends_valid_http_request():
+    # These values aren't actual creds but resemble values used in production
+    ge_cloud_base_url = "https://app.test.greatexpectations.io"
+    ge_cloud_organization_id = "229616e2-1bbc-4849-8161-4be89b79bd36"
+    ge_cloud_access_token = "d7asdh2efads9afah2e0fadf8eh20da8"
```

Comment:
Is there a reason to not use the existing global fixtures for these values? Just figure keep things DRY if we need to change them in the future.

Reply:
I know @billdirks has pushed back on this a bit in the past and I see merits with both approaches - as these values are simple placeholders, do we need to couple additional dependencies here?

Good either way but just explaining the approach here.

Reply:
Either is fine. I am thinking more if the url changes (doubtful) or we break something in CI and accidentally start sending usage events from tests to prod. If there are consistent ids to ignore that would be easier. But I'm probably being too cautious.